pkg/controller/node/node_controller.go

```diff
@@ -461,8 +462,15 @@ func (r *ReconcileNode) Reconcile(request reconcile.Request) (reconcile.Result,
 	// TODO: find a way to fill the segmentPort by using the return value from searchNodePortByVcNameAddress then we won't need to invoke the Get API again.
 	sl := strings.Split(*segmentPort.Path, "/")
 	segmentId := sl[len(sl)-1]
-	portsClient := segments.NewDefaultPortsClient(nsxClients.PolicyConnector)
-	*segmentPort, err = portsClient.Get(segmentId, *segmentPort.Id)
+	portsClient := infra_segment.NewDefaultPortsClient(nsxClients.PolicyConnector)
```

Comment:
Move line 465 to 472?
Please add the path pattern for infra segment in comments as well

Reply:
> Move line 465 to 472?
Previous this line locates at 472, but it reported an error that portsClient is undefined, so I moved it to 465
> Please add the path pattern for infra segment in comments as well
Done

Reply:
Move 465 to 472, and in 470 define portsClient as well
portsClient := tier1_segment.NewDefaultPortsClient(nsxClients.PolicyConnector)